src/value.sol

```diff
@@ -0,0 +1,25 @@
+/// value.sol - minimal ValueLike implementation for verification
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.6;
+
+contract Value {
```

Comment:
n.b. this has a different storage layout to a DSValue.

Reply:
see e.g. https://dapp.ci/k-dss/a8c48ed1c8546a39c21a/#DSValue_peek() for DSValue specs.

Reply:
Have updated it to use DSValue, just waiting on the fv build queued here https://buildbot.dapp.ci/#/builders/8